examples/notebooks/beam-ml/bigtable_enrichment_transform.ipynb

```diff
@@ -624,6 +642,44 @@
         "  return beam.Row(**enriched)"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "To provide a `lambda` function for using a custom join with the enrichment transform, see the following example.\n",
+        "\n",
+        "```\n",
+        "with beam.Pipeline() as p:\n",
+        "  output = (p\n",
+        "            ...\n",
+        "            | \"Enrich with BigTable\" >> Enrichment(bigtable_handler, join_fn=custom_join)\n",
+        "            | \"RunInference\" >> RunInference(model_handler)\n",
+        "            ...\n",
+        "            )\n",
+        "```"
+      ],
+      "metadata": {
+        "id": "fe3bIclV1jZ5"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Because the enrichment transform make API calls to the remote service, use the `timeout` parameter to specify a timeout duration of 10 seconds:\n",
```

Comment:
```suggestion
        "Because the enrichment transform makes API calls to the remote service, use the `timeout` parameter to specify a timeout duration of 10 seconds:\n",
```